index.bs

```diff
@@ -12031,7 +12031,22 @@ class Vec3 {
 
 <h2 id="changelog">
 Change Log</h2>
-<h3 id="changestart">
+<h3 id="changestart1">
+Since Working Draft of 19 June 2018</h3>
+* Update implementation-report.html
+* Widen range for detune values
```

Comment:
This is actually restricting the range. Previously it was +/- inf (exclusive), the we reduced it to a much smaller range, and but then made it wider to cover more.  So compared to before, it's a restriction of the range.